lib/browser/api/web-frame-main.ts

```diff
@@ -7,15 +7,23 @@ WebFrameMain.prototype.send = function (channel, ...args) {
     throw new Error('Missing required channel argument');
   }
 
-  return this._send(false /* internal */, channel, args);
+  try {
+    return this._send(false /* internal */, channel, args);
+  } catch (e) {
```

Comment:
I don't think this is an ideal fix. I don't think we want to swallow all exceptions like for example:
https://github.com/electron/electron/blob/cf0959f8340208f24baeaee29f381c2a96276c8c/shell/browser/api/electron_api_web_frame_main.cc#L171-L175

IMO we should instead avoid throwing in the first place in `WebFrameMain::Send`, `WebFrameMain::PostMessage`, and perhaps `WebContents::MessageTo`. We could do that by replacing the `CheckRenderFrame()` call with something that just checks the status instead of also throwing.

cc @VerteDinde

Reply:
Thanks @poiru! Just saw your comments here, will open a new PR with those improvements. We're in the middle of a December quiet period, so we may not have a new release with the fix until January, but will work on getting that fix ready.

Reply:
Perfect, thank you @VerteDinde!

Reply:
@VerteDinde Friendly reminder about this!